force-app/main/default/classes/PMT_RefundController.cls

```diff
@@ -133,6 +131,8 @@ public with sharing class PMT_RefundController {
             return;
         }
 
+        refundService.updateParentOpportunity(originalPayment, refundAmount);
```

Comment:
This method will be removed in the next WI when implementing Allocation and opportunity update inside trigger.